frontend/src/ui/LandingPage.tsx

```diff
@@ -0,0 +1,62 @@
+import { Link } from '@material-ui/core';
+import React from 'react';
+import { PageContainer, Row, Section, ShadowButton } from './Common';
+import Logo from './Logo';
+import { Body, Title } from './Typography';
+
+export interface LandingPageProps {
+  title: string;
+  body: string;
+}
+
+const LandingPage = () => {
+  return (
+    <PageContainer>
+      <TopBar />
+      <Section
+        style={{
+          width: '600px',
+          margin: 'auto',
+          marginTop: '100px',
+        }}
+      >
+        <Title>JobHopper, A Code For Boston Project</Title>
```

Comment:
Ed updated the text for this page, refer to the latest mocks: https://www.figma.com/proto/fXQirNUcSUBC5KTJ7hYBO6/JobHopper-interface-2?node-id=2%3A2&scaling=min-zoom